models/classes/ServiceProvider/LtiServiceProvider.php

```diff
@@ -259,5 +262,17 @@ public function __invoke(ContainerConfigurator $configurator): void
                     param('rolesAllowed')
                 ]
             );
+
+        $services
+            ->set(AuthoringAsToolLtiConfigProvider::class)
+            ->decorate(AuthoringAsToolConfigProviderInterface::class)
```

Comment:
Instead of decorating I would suggest to overload implementation of AuthoringAsToolConfigProviderInterface*

* [recommended name change](https://github.com/oat-sa/tao-core/pull/3983/files/26c02bb41efd98dec0ed050a95ffb9784bd1071c#r1537934883)